provisioning/baremetal_config.go

```diff
@@ -95,6 +95,33 @@ func getProvisioningOSDownloadURL(config *metal3iov1alpha1.ProvisioningSpec) *st
 	return nil
 }
 
+// Check whether the PreProvisionOSDownloadURLs are set. If yes, we
+// construct a comma-separated list of RHCOS live images and return it
+func getPreProvisioningOSDownloadURLs(config *metal3iov1alpha1.ProvisioningSpec) []string {
+	var liveURLs []string
+	if config.PreProvisioningOSDownloadURLs.InitramfsURL != "" {
+		liveURLs = append(liveURLs, config.PreProvisioningOSDownloadURLs.InitramfsURL)
```

Comment:
Should this be IsoURL?

Reply:
Yes. Correct. I meant IsoURL in both places.